src/main/sessions.ts

```diff
@@ -213,7 +247,7 @@ class JupyterLabSessions extends EventEmitter implements ISessions, IStatefulSer
             
         });
 
-        ipcMain.once(AppIPC.LAB_READY, () => {
+        ipcMain.once('lab-ready', () => {
```

Comment:
Would it be better to replace this hard coded string with a constant, similar to what previously existed?